source/playbooks/CIS140/ssmdocs/CIS140_2.2.1.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { Construct } from 'constructs';
+import { PlaybookProps } from '../../SC/lib/control_runbooks-construct';
+import { ControlRunbookDocument } from '../../SC/ssmdocs/control_runbook';
+import { EnableEbsEncryptionByDefaultDocument } from '../../SC/ssmdocs/SC_EC2.7';
+
+export function createControlRunbook(stage: Construct, id: string, props: PlaybookProps): ControlRunbookDocument {
+  return new EnableEbsEncryptionByDefaultDocument(stage, id, { ...props, controlId: '3.1' });
```

Comment:
wrong control ID. I think you also have to add this to `source/playbooks/CIS140/bin/cis140.ts` and `source/playbooks/CIS140/lib/control_runbooks-construct.ts` in order for the doc to make it into the template and for the eventbridge rule to be created.